<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiveOrderCheckWidget</class>
 <widget class="QWidget" name="ReceiveOrderCheckWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="backButton">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>18</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	border-image: url(:/orderGoodsImage/leftArrow.png);
}

QPushButton:pressed{
	border-image: url(:/orderGoodsImage/leftArrow_press.png);
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:16px;
color:#ffffff;</string>
           </property>
           <property name="text">
            <string>返回送回单列表</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#999999;</string>
           </property>
           <property name="text">
            <string>送货单号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="orderIdValueLabel">
           <property name="styleSheet">
            <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#999999;</string>
           </property>
           <property name="text">
            <string>21555558888</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1</width>
             <height>12</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid #6c6c6c;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#999999;</string>
           </property>
           <property name="text">
            <string>供应商：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="providerNameLabel">
           <property name="styleSheet">
            <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#999999;</string>
           </property>
           <property name="text">
            <string>兄弟水果供应商</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1</width>
             <height>12</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid #6c6c6c;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="orderTimeLabel">
           <property name="styleSheet">
            <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#999999;</string>
           </property>
           <property name="text">
            <string>2016.02.12 12:20</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ReceiveOrderGoodsWidget" name="receiveOrderGoodsWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReceiveOrderGoodsWidget</class>
   <extends>QWidget</extends>
   <header>ReceiveOrderGoodsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
